pkg/lib/resourcescope/store_resource.go

```diff
@@ -173,7 +173,7 @@ type storeListResourceResult struct {
 
 func (s *Store) ListResources(ctx context.Context, options *ListResourcesOptions, pageArgs graphqlutil.PageArgs) (*storeListResourceResult, error) {
 	q := s.selectResourceQuery("r").
-		OrderBy("r.uri ASC")
+		OrderBy("r.created_at DESC")
```

Comment:
ASC. The index you created is for ASC.

Reply:
https://www.postgresql.org/docs/current/indexes-ordering.html#:~:text=By%20default%2C%20B%2Dtree%20indexes,BY%20x%20ASC%20NULLS%20LAST%20).

betree indexes can be scanned backward, therefore it should work in both direction. Or we need two indexes in queries which allow sorting in both directions.